lib/ttdrest/concerns/my_reports.rb

```diff
@@ -10,23 +10,28 @@ module MyReports
       def get_my_reports(report_date, options = {})
         advertiser_id = self.advertiser_id || options[:advertiser_id]
         path = '/myreports/reportexecution/query/advertisers'
-        content_type = 'application/json'
         report_data = {}.tap do |search_params|
           search_params['AdvertiserIds'] = [advertiser_id]
           search_params['ReportDateUTC'] = report_date.strftime("%Y-%m-%d")
           search_params['ReportScheduleNameContains'] = options[:report_schedule_name_contains] if options[:report_schedule_name_contains]
-          search_params['ExecutionStates'] = options[:execution_states].present? ? options[:execution_states] : ["Complete"]
-          search_params['ExecutionSpansStartDate'] = options[:execution_spans_start_date].present? ? options[:execution_spans_start_date] : report_date.strftime("%Y-%m-%dT00:00:00.0000000+00:00")
-          search_params['ExecutionSpansEndDate'] = options[:execution_spans_end_date].present? ? options[:execution_spans_end_date] : (report_date + 1.day).strftime("%Y-%m-%dT00:00:00.0000000+00:00")
-          search_params['PageStartIndex'] = options[:page].present? ? options[:page] : 0
-          search_params['PageSize'] = options[:page_size].present? ? options[:page_size] : 1
+          search_params['ExecutionStates'] = present_or_default(options[:execution_states],["Complete"])
+          search_params['ExecutionSpansStartDate'] = present_or_default(options[:execution_spans_start_date], report_date.strftime("%Y-%m-%dT00:00:00.0000000+00:00"))
+          search_params['ExecutionSpansEndDate'] = present_or_default([:execution_spans_end_date], (report_date + 1.day).strftime("%Y-%m-%dT00:00:00.0000000+00:00"))
+          search_params['PageStartIndex'] = present_or_default(options[:page], 0)
```

Comment:
keywords would help on this method

Reply:
I thought about this, but let the line lengths get in my head. I was hoping my method name defined it well enough.